lm75bd/lm75bd.h

```diff
@@ -21,6 +21,9 @@
 #define LM75BD_DEFAULT_OT_THRESH 80.0f
 #define LM75BD_DEFAULT_HYST_THRESH 75.0f
 
+// Multiplier to convert temp. register value to Celsius
+#define LM75BD_TEMP_C_MULTIPLIER 0.125
```

Comment:
this should go in the .c file since it's only used internally within the driver and doesn't need to be accessible to other modules that try to include the driver through this .h file